packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -99,6 +99,28 @@ const cli = {
           return undefined;
         },
       },
+      seleniumServiceUrl: {
```

Comment:
We tend to keep these options alphabetized. Can you put them in alphabetically orders here, [here](https://github.com/cerner/terra-toolkit/pull/691/files#diff-86631c8f3fb52653295ad5b333c6724f3f393a9bec4a477083220d77c6a4c700R19-R20), and [here](https://github.com/cerner/terra-toolkit/pull/691/files#diff-bd6f2cecd6d3c745845a151947012865dd0ad4b869000d4478ffd2bbe28532ebR49-R50)?

Reply:
I just noticed I put seleniumServiceUrl before seleniumServicePort because that seemed like the intuitive order for those. Do you want me to reverse them? Or should I change Url to seleniumServiceHost to be like externalHost?

Reply:
Sure if you don't mind just to be consistent.

Reply:
@benbcai reverse them and keep Url or switch to Host?

Reply:
Sorry for my unclear response. Let's keep `seleniumServiceUrl` and just swap the ordering between seleniumServiceUrl and seleniumServicePort.